content/blog/2023-09-27-epidatr.Rmd

```diff
@@ -0,0 +1,94 @@
+---
+title: New package released! `epidatr`, an R client for Delphi Epidata API
+author: Dmitry Shemetov, David Weber
+date: 2023-09-27
+tags:
+  - r
+  - epidata
+  - covidcast
+authors:
+  - dmitry
+  - david
+heroImage: blog-lg-epidatr.jpg
+heroImageThumb: blog-thumb-epidatr.jpg
+summary: |
+  `epidatr` is designed to streamline the downloading and usage of data from the [Delphi Epidata API](https://cmu-delphi.github.io/delphi-epidata/). It provides a simple R interface to the API, including functions for downloading data, parsing the results, and converting the data into a tidy format.
+ 
+output:
+  blogdown::html_page:
+    toc: true
+---
+
+The [Delphi Epidata API](https://cmu-delphi.github.io/delphi-epidata/) provides real-time access to epidemiological surveillance data for influenza, COVID-19, and other diseases from both official government sources such as the [Center for Disease Control (CDC)](https://www.cdc.gov/datastatistics/index.html) and [Google Trends](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/google-symptoms.html) and private partners such as [Facebook](https://delphi.cmu.edu/blog/2020/08/26/covid-19-symptom-surveys-through-facebook/) and [Change Healthcare](https://www.changehealthcare.com/). It is built and maintained by the Carnegie Mellon University [Delphi research group](https://delphi.cmu.edu/).
+
```

Comment:
We need some actual links to epidatr!!!
```suggestion

Today we introduce the R package "`epidatr`", available [on CRAN](https://cloud.r-project.org/web/packages/epidatr/index.html), with the source and development [on github](https://github.com/cmu-delphi/epidatr).

```

Reply:
Links to the tool we're talking about? That's crazy!